catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: workflows
+  title: Workflows
```

Comment:
Could we give this a more globally unique name perhaps `Python Workflows` & `python-workflows` like the repository? Or is `Workflows` the new name?

Reply:
`workflows` is the name for the [pypi](https://pypi.org/project/workflows/) and [conda-forge](https://anaconda.org/conda-forge/workflows) packages, but happy to change the name for the catalogue if you think it would be better to match the repository name.

Reply:
I'd stick with `workflows` if that is the case